components/picker/index.css

```diff
@@ -10,409 +10,533 @@ OF ANY KIND, either express or implied. See the License for the specific languag
+    --spectrum-picker-spacing-picker-to-popover: var(--spectrum-component-to-menu-small);
   }
 }
 
 .spectrum-Picker--sizeM {
-  @remapvars {
-    find: /--spectrum-picker-m(.*)/;
-    filter: color;
-    replace: --spectrum-picker$1;
+  --spectrum-picker-font-size: var(--spectrum-font-size-100);
+  --spectrum-picker-block-size: var(--spectrum-component-height-100);
+  --spectrum-picker-spacing-text-to-icon: var(--spectrum-text-to-visual-100);
+  --spectrum-picker-spacing-text-to-alert-icon-inline-start: var(--spectrum-field-text-to-alert-icon-medium);
+  --spectrum-picker-spacing-icon-to-disclosure-icon: var(--spectrum-picker-visual-to-disclosure-icon-medium);
+  --spectrum-picker-spacing-label-to-picker-quiet: var(--spectrum-field-label-to-component-quiet-medium);
+
+  & + .spectrum-Popover--bottom.is-open {
+    --spectrum-picker-spacing-picker-to-popover: var(--spectrum-component-to-menu-medium);
   }
 }
 
 .spectrum-Picker--sizeL {
-  @remapvars {
-    find: /--spectrum-picker-l(.*)/;
-    filter: color;
-    replace: --spectrum-picker$1;
+  --spectrum-picker-font-size: var(--spectrum-font-size-200);
+  --spectrum-picker-block-size: var(--spectrum-component-height-200);
+  --spectrum-picker-spacing-text-to-icon: var(--spectrum-text-to-visual-200);
+  --spectrum-picker-spacing-text-to-alert-icon-inline-start: var(--spectrum-field-text-to-alert-icon-large);
+  --spectrum-picker-spacing-icon-to-disclosure-icon: var(--spectrum-picker-visual-to-disclosure-icon-large);
+  --spectrum-picker-spacing-label-to-picker-quiet: var(--spectrum-field-label-to-component-quiet-large);
+
+  & + .spectrum-Popover--bottom.is-open {
+    --spectrum-picker-spacing-picker-to-popover: var(--spectrum-component-to-menu-large);
   }
 }
 
 .spectrum-Picker--sizeXL {
-  @remapvars {
-    find: /--spectrum-picker-xl(.*)/;
-    filter: color;
-    replace: --spectrum-picker$1;
+  --spectrum-picker-font-size: var(--spectrum-font-size-300);
+  --spectrum-picker-block-size: var(--spectrum-component-height-300);
+  --spectrum-picker-spacing-text-to-icon: var(--spectrum-text-to-visual-300);
+  --spectrum-picker-spacing-text-to-alert-icon-inline-start: var(--spectrum-field-text-to-alert-icon-extra-large);
+  --spectrum-picker-spacing-icon-to-disclosure-icon: var(--spectrum-picker-visual-to-disclosure-icon-extra-large);
+  --spectrum-picker-spacing-label-to-picker-quiet: var(--spectrum-field-label-to-component-quiet-extra-large);
+
+  & + .spectrum-Popover--bottom.is-open {
+    --spectrum-picker-spacing-picker-to-popover: var(--spectrum-component-to-menu-extra-large);
   }
 }
 
-.spectrum-Picker {
-  /* todo: remove when fixed in DNA */
-  --spectrum-picker-texticon-min-width: var(--spectrum-global-dimension-size-400);
-
-  /* Todo: fix in DNA, should have been zero */
-  --spectrum-picker-texticon-disabled-border-size: 0;
+/* Windows high contrast mode */
+@media (forced-colors: active) {
+  .spectrum-Picker {
+    --highcontrast-picker-focus-indicator-color: Highlight;
```

Comment:
This should be CanvasText.

` --highcontrast-picker-focus-indicator-color: CanvasText`